scripts/augment.js

```diff
@@ -0,0 +1,198 @@
+const fs = require('fs')
+const lang = process.argv[2]
+if (!lang) {
+  const files = fs.readdirSync('dist')
+  const langs = files.map(tag => tag.match(/-(.+)\./)[1])
+  console.log(`Please provide a language tag: ${langs.join(', ')}`)
+} else {
+  start()
+}
```

Comment:
The only hesitation I have is with the lang attribute here. I think it is safe to just default to `en-US` since platform data is unlikely to have internationalized data afaik. Perhaps just say `augument-en`?

This is non-blocking and looks good to me. You should be able to merge, right? Then I can publish a release.

Reply:
That sounds great to me, thanks!